documentation/book/snip-images.adoc

```diff
@@ -8,15 +8,16 @@ a|
 * {DockerOrg}/kafka:{DockerTag}-kafka-2.0.1
 * {DockerOrg}/kafka:{DockerTag}-kafka-2.1.0
 * {DockerOrg}/kafka:{DockerTag}-kafka-2.1.1
+* {DockerOrg}/kafka:{DockerTag}-kafka-2.2.0
 
 a|
-{ProductName} image for running Kafka, including:
+AMQ Streams image for running Kafka, including:
```

Comment:
Why AMQ Streams here?

Reply:
Good that you noticed this. I didnt changed that manually - there seems to be bug in one of the scripts generating this. I fixed it now (in the script + regenerated the docs).